src/main/java/seedu/address/model/person/LicencePlate.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's car licence plate in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLicencePlate(String)}
+ */
+public class LicencePlate {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Licence plates should be in the format Sxx####y which follows the following constraints:\n"
+            + "1. S - starts with the letter S, which stands for private vehicle\n"
+            + "2. x - Alphabetical series (all except 'I' and 'O')\n"
+            + "3. #### – Numerical series (from 1 to 9999, without leading zeroes)\n"
+            + "4. y – Checksum letter (all except 'F', 'I', 'N', 'O', 'Q', 'V' and 'W'";
```

Comment:
Seems like this would account for all Singapore vehicles.

Reply:
I referenced [this](https://en.wikipedia.org/wiki/Vehicle_registration_plates_of_Singapore) and there seems to be A LOT more car plates so I don't think we can account for all